espnet2/bin/svs_inference.py

```diff
@@ -94,11 +95,16 @@ def __init__(
     def __call__(
         self,
         text: torch.Tensor,
-        score: Optional[torch.Tensor],
-        durations: Union[torch.Tensor, np.ndarray] = None,
         singing: torch.Tensor = None,
+        label_lab: Optional[torch.Tensor] = None,
+        midi_lab: Optional[torch.Tensor] = None,
+        tempo_lab: Optional[torch.Tensor] = None,
+        beat_lab: Optional[torch.Tensor] = None,
+        label_xml: Optional[torch.Tensor] = None,
```

Comment:
How about change xml into `score`?

Reply:
The name score is used otherwhere for another meaning. I haven't thought of a better name yet.

Reply:
I see. Documentation could probably mitigate the issue.